Source/lua/lua.cpp

```diff
@@ -113,8 +123,8 @@ void LuaInitialize()
 	    "render", LuaRenderModule(lua),
 	    "message", [](std::string_view text) { EventPlrMsg(text, UiFlags::ColorRed); });
 
-	RunScript("lua/init.lua");
-	RunScript("lua/user.lua");
+	RunScript("lua" DIRECTORY_SEPARATOR_STR "init.lua", /*optional=*/false);
+	RunScript("lua" DIRECTORY_SEPARATOR_STR "user.lua", /*optional=*/true);

```

Comment:
Since we're loading Lua these scripts using `FindAsset()`, I think we should be using backslash.

Reply:
Ah, that's right, fixed